src/praisonai-mobile/app/app.css

```
@@ -0,0 +1,204 @@
+  --safe-area-inset-left: env(safe-area-inset-left, 0px);
+
+  --ground: #ffffff;
+  --panel: #f5f6f8;
+  --ink: #14171c;
+  --soft: #5b636e;
+  --rule: #e2e5ea;
+  --accent: #1f7a63;
+  --bad: #a6413c;
+  --warn: #b34700;
+  --radius: 12px;
+  --sans: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)